examples/js-demo/RelatedItem.tsx

```diff
@@ -1,3 +1,4 @@
+/** @jsxRuntime classic */
```

Comment:
why do we need to add this, can you give us a little bit more context, in case we encounter more instances of this error?

Reply:
It is due to the way the `recommend-vdom` is setup with `preact` & `preact/hooks`. 
We basically need to tell the tooling (babel) how to interpret JSX syntax and transform it into plain JavaScript on the output. 

`/** @jsx h */` -> Tells Babel to transform JSX into Preact `h()` calls instead of `React.createElement`

By default Babel (after v7.9.0) has an "automatic runtime" where it will import all the functions it will compile the JSX to. Since we're not using React but Preact which sets back the old behavior of allowing us to customize those JSX transformation functions.  

https://preactjs.com/guide/v8/getting-started/
https://babeljs.io/docs/babel-plugin-transform-react-jsx#react-classic-runtime 

PS: Thanks for the good question I had to go back and look some references up!

Reply:
thanks a lot for the detailed explanation 

approve